src/pleo-radar.json

```
@@ -201,5 +201,26 @@
       "quadrant": "languages & frameworks",
       "isNew": "FALSE",
       "description": ""
+    },
+    {
+      "name": "Go",
+      "ring": "adopt",
+      "quadrant": "languages & frameworks",
+      "isNew": "FALSE",
+      "description": ""
+    },
+    {
+      "name": "Javascript",
```

```suggestion
      "name": "JavaScript",
```

The same should probably be done for TypeScript above